.gitignore

```diff
@@ -1,2 +1,5 @@
 target
 work
+.project
```

Comment:
Comment: we should consider applying a standard .gitignore to all our projects (exclude IDE files, build files, work, and JS node stuff).